### PR TITLE
Removing test coverage for shared code.

### PR DESCRIPTION
**Pull request by kwoodson**

### CODE REVIEW

File: roles/lib_openshift/src/generate.py [hunk 1]
```diff
@@ -72,6 +76,11 @@ def generate(parts):
                 if idx in [0, 1] and 'flake8: noqa' in line or 'pylint: skip-file' in line:  # noqa: E501
                     continue
 
+                for skip in SKIP_COVERAGE_PATTERN:
```
Comment (kwoodson) [inline roles/lib_openshift/src/generate.py]: The main changes here are skipping the tests for shared code.

**Comment by ashcrow** [comment]
aos-ci-test

**Comment by ashcrow** [comment]
[test]

**Comment by ashcrow** [comment]
[merge]

**Comment by sdodson** [comment]
[merge]

**Pull request merged**